nix/flake-modules/checks.nix

```diff
@@ -0,0 +1,47 @@
+{ self, inputs, ... }:
+{
+  perSystem =
+    { self'
+    , pkgs
+    , lib
+    , ...
+    }:
+    let
+      inherit (inputs) nixpkgs;
+      moduleTest =
+        (nixpkgs.lib.nixos.runTest {
+          hostPkgs = pkgs;
+          defaults.documentation.enable = false;
+          imports = [
+            {
+              name = "teslamate";
+              nodes.server = {
+                imports = [ self.nixosModules.default ];
+                virtualisation.cores = 4;
+                virtualisation.memorySize = 2048;
```

Comment:
I increased the defaults here cuz the default test VM only gets 1 core and 1024MB of memory and that seemed to be causing some trouble for postgres to startup without systemd timing it out.

Reply:
Nice tweak, ty!